src/app/_components/PillButton/PillButton.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
```

Comment:
🟠 `PillButton` => `component` ? so structure will be
`PillButton/component.js`
`PillButton/styles.js`